specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -728,8 +728,11 @@
       ],
       "properties": {
         "criteria": {
-          "$ref": "#/definitions/Criteria",
-          "description": "Severity of the alert"
+          "type": "array",
```

Comment:
Is the type "LogToMetricAction" sent in by the user or is it something that gets returned form the server side and the user treats as a read only type?

This seems like a breaking change.. has the SDK without these changes already been released?

Reply:
LogToMetricAction is sent by user in the request payload, The criteria filed was always supposed to be arrray but sdk got released with incorrect definition.

https://github.com/Azure/azure-rest-api-specs/issues/5326 has been opened for same.